### PR TITLE
Fixes and improvements

### PR DESCRIPTION
**Pull request by ByFox213**
Added a function with asynchronous call as well as an example
Fixed a lot of errors in code readability and bugs

I didn't like the code, so I did it.

https://peps.python.org/

### CODE REVIEW

**Comment by ByFox213** [comment]
I forgot to add a check for the number of buttons.

**Comment by Senophyx** [comment]
Hello, thank you for your pull request. I will reviewing your pull ASAP.

*Sorry for the late response. I've been very busy lately.*

File: discordrpc/utils.py
Comment (Senophyx) [inline discordrpc/utils.py:1]: Is there any particular reason why you changed the `timestamp` variable to the `get_timestamp` function?

File: discordrpc/button.py
Comment (Senophyx) [inline discordrpc/button.py:1]: For some reason, I think it's going to be more complicated. Wouldn't it be easier for users to use variables?

**Comment by Senophyx** [comment]
Please review my comments and resolve all conflicting files before i merge your pull request. Thank you.

**Comment by ByFox213** [comment]
There are conflicts here that I don't understand.

**Comment by ByFox213** [comment]
He's complaining about the gaps

**Comment by ByFox213** [comment]
As for the conflict with presence.py, I moved these lines to another file

**Comment by ByFox213** [comment]
After trying so many times, I figured out how to work with the conflict system. actually because of the conflicts, some of my changes were broken. Check if I have changed what you asked for and you can finish with this miracle.

**Review by Senophyx — DISMISSED** [review]
Approve

**Comment by Senophyx** [comment]
On testing

**Comment by Senophyx** [comment]
For some reason, no output is displayed in the terminal even though the `output` argument is `True`

**Pull request closed without merging**